packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md

```diff
@@ -0,0 +1,224 @@
+
+> **Unsupported scopes:** <br>
+> * `offline_access` scope isn't available since refresh tokens aren't supported for apps published in OIN.
+> * `groups` claims as described in the supported [Scopes](/docs/reference/api/oidc/#scopes) section isn't supported.
+
+<ApiAmProdWarning />
+
+## Logo guidelines
+
+The app logo that you submit to the OIN Manager in the **General Settings** > **App icon** field must conform to the following guidelines:
+
+* The logo file size must be less than one MB.
+* The recommended logo dimension has a 1:1 (square) aspect ratio.
+
+   > **Note:** OIN app icons are placed in a 200 x 200 pixels grid.
+
+* The recommended logo contains only a logomark and not a logotype (such as a company icon and not the company name).
+
+   > **Note:** The company name is already listed in plain text in the OIN catalog, so adding a company name image is redundant.
+
+* Avoid logos with both an icon and a wordmark.
+* Avoid logos with the trademark (:tm:) symbol.
+* The recommended logo is an image in PNG format with a transparent background.
+    * GIF and JPEG/JPG formats are also acceptable
+    * A colored background is acceptable if it's a part of the logo color scheme
+
+### Comply with the logo guidelines
+
+If your logo doesn't meet the OIN requirements, the following are suggestions for you to conform to the guidelines:
+
+* If your logo isn't in one of the acceptable formats, ask your design team to convert your existing logo to the preferred PNG format. If you have difficulty with converting your image to an acceptable format, contact the OIN team.
+
+* If your logo isn't square, consider using your website favicon. Alternatively, you can use the first letter of your app wordmark and convert it to a square image.
+
+* You can use a slightly non-square icon, but take into account the scaling that occurs when your logo is fitted into a 200 x 200 pixels space. If you have additional logo questions, contact the OIN team.
+
+<div class="full">
+
+![OIN acceptable logos](/img/oin/oin-logo-size-guide.png)
+
+</div>
+
+### Example of acceptable logos
+
+<div class="full">
+
+![OIN acceptable logos](/img/oin/oin-logo-guide.png)
+
+</div>
+
+## App description guidelines
+
+An app description is displayed on your integration tile in the OIN catalog. You can specify your app description in the **General Settings** > **App description** field from the OIN Manager. The app description content appears in your Integration detail page, under the **Overview** header in the OIN catalog. See the [OIN catalog](https://www.okta.com/integrations/) for examples of existing app descriptions in their associated Integration page.
+
+Prepare an app description that describes what your integration offers and the supported features. You can also provide value-added messaging for your integration. Your description needs to be less than 1024 characters.
+
+## Use case guidelines
```

Comment:
@tovamarkowitz-okta, side note: now that this is use case and not category of SaaS app, I think we could stop asking for it in the OIN Manager? API Service integrations might be an exception but maybe we can work through that with some other solution. 99% are SSO or provisioning and we should know that based on the type of integration they build.